src/controllers/comment.js

```diff
@@ -0,0 +1,32 @@
+/* eslint-disable require-jsdoc */
+import models from '../models';
+
+const { Comment } = models;
+
+export default class CommentController {
+  /**
+     * @param {Object} req the user's comment
+     * @param {Object} res success or error object
+     * @returns {Object} details of the posted comment
+     * @description Post a new comment about a trip request made
+     */
+  static async postComment(req, res) {
+    const { commentBody } = req.body;
+    try {
+      const userId = req.user.id;
+      const { requestId } = req.params;
+
+      const newComment = await Comment.create({
+        commentBody, requestId, userId
+      });
+      res.status(201).json({
+        status: 'success',
+        data: {
+          newComment: newComment.commentBody
```

Comment:
I feel the response data should also contain the details of the user who posted the comment.